### PR TITLE
brew.md: add Linuxbrew version

### PR DESCRIPTION
**Pull request by waldyrious**
See https://github.com/Linuxbrew/homebrew-core/pull/1472 for context.

### CODE REVIEW

**Comment by MarloweW** [comment]
Maybe we could add `brew info -f {{formula}}`, it's useful to find out files contained in a formula.

**Comment by waldyrious** [comment]
@MarloweW this would be applicable to the osx version as well, right? If so, I'd rather do that in a separate PR.

**Comment by MarloweW** [comment]
@waldyrious  I must have misremember it. 

`brew info {{formula}}` will just print description of formula and which directory current version of this formula is installed, `-f` has no effect on this command. To list all files provided by a formula, we should use `brew ls {{--verbose|-v}} {{formula}}`. And yes, they both are applicable to osx version and Linux version, checked source code.

A separate PR would be better.

File: pages/linux/brew.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+`brew search {{text}}`
+
+- Install formula:
+
+`brew install {{formula}}`
+
+- List installed formulae [with matching name]:
+
+`brew list {{[text]}}`
```
Comment (agnivade) [inline pages/linux/brew.md]: umm .. why the square brackets ?

Comment (waldyrious) [inline pages/linux/brew.md]: This page is identical to osx/brew.md, except for the description. I'd rather make any changes in another PR. As for this particular notation, I suppose it means that the parameter is optional; without, the full list would be printed. But I agree it isn't super-intuitive. Let's change it in a separate PR.

Comment (agnivade) [inline pages/linux/brew.md]: ok
